pets/mongodb/README.md

```diff
@@ -0,0 +1,4 @@
+# TODO
```

Comment:
Can you write down some steps how to setup kubernetes environment (if necessary) and how to run ``mongodb`` petset? Something similar to [1].

[1] https://github.com/kubernetes/contrib/blob/master/pets/zookeeper/README.md